ocs_samples/basic_samples/SDS/JavaScript/NodeJs/readme.md

```diff
@@ -324,20 +383,119 @@ restCall({
 });
 ```
 
--  parameters are the SdsStream Id and the starting and ending index
-   values for the desired window Ex: For a time index, request url
-   format will be
-   "/{streamId}/Data?startIndex={startTime}&endIndex={endTime}
+- *start* and *end* (inclusive) represent the indices for the retrieval. 
+- The namespace ID and stream ID must be provided to the function call.
+- A JSON object containing a list of the found values is returned. 
+ Ex: For a time index, request url format will be
+    "/{streamId}/Data?startIndex={startTime}&endIndex={endTime}
+
+
+Here is how it is called:
+
+```js
+client.getWindowValues(tenantId, sampleNamespaceId, sampleStreamId, 0, 180);
+```
+
+<h5>Get Range Values</h5>
+
+``getRangeValues`` is used for retrieving a specified number of events from 
+a starting index. This method in ``SdsClient`` allows retrieval of a range 
+of values from a start index. The starting index is the ID of the 
+``SdsTypeProperty`` that corresponds to the key value of the WaveData type. 
+Here is the request:
+
+```js
+restCall({
+    url: this.url + this.streamsBase.format([tenantId, namespaceId]) + this.getRangeValuesBase.format([streamId, start, skip, count, reverse, boundaryType, streamView]),
+    method: 'GET',
+    headers: this.getHeaders()
+});
+```
+
+- *skip* is the increment by which the retrieval will happen.
+- *count* is how many values you wish to have returned.
+- *reverse* is a boolean that when ``true`` causes the retrieval to work 
+backwards from the starting point.
+- *boundary\_type* is a ``SdsBoundaryType`` value that determines the 
+behavior if the starting index cannot be found. Refer the to the 
+[SDS documentation](https://ocs-docs.osisoft.com/Documentation/SequentialDataStore/Data_Store_and_SDS.html) for more information about SdsBoundaryTypes.
+
+The ``getRangeValues`` method is called as shown :
+
+```js
+client.getRangeValues(tenantId, sampleNamespaceId, sampleStreamId, "1", 0, 3, "False", sdsObjs.sdsBoundaryType.ExactOrCalculated);
+```
+
+<h5>Getting Table Form</h5>
```

Comment:
I would just call this section Table Format or "Getting data in table format." It may be better just to tack these on to the sections about GetWindowValues and GetRangeValues because they are just overloads for the same method in SDS.